Mathlib/Combinatorics/SimpleGraph/Basic.lean

```diff
@@ -1306,6 +1297,12 @@ lemma le_comap_of_subsingleton (f : V → W) [Subsingleton V] : G ≤ G'.comap f
 lemma map_le_of_subsingleton (f : V ↪ W) [Subsingleton V] : G.map f ≤ G' := by
   rw [map_le_iff_le_comap]; apply le_comap_of_subsingleton
 
+/-- Given a family of vertex types indexed by `ι`, pulling back from `⊤ : SimpleGraph ι`
+yields the complete multipartite graph on the family.
+Any graph may be regarded as a subgraph of one of these, with sufficiently many parts. -/
```

Comment:
This last sentence is true, and I see you're going for parallelism with the `completeBipartiteGraph` docstring, but it's not very helpful to know immediately (it could be a nice theorem over in the file on graph colorings though -- given a graph coloring, you can create a multipartite graph out of the color classification function, then identify the original graph as a subgraph of the `completeMultipartiteGraph`. Not in this PR though.)

This would be very helpful to the reader:
```suggestion
Two vertices are adjacent if and only if their indices are not equal. -/
```